torch/jit/__init__.py

```diff
@@ -941,13 +941,31 @@ def env(key):
     return env
 
 
+def _make_strong_submodule(field, module, parent):
+    if field not in parent._modules:
```

Comment:
how does this happen? There are nn.Modules in module that are not in the _modules list?

Reply:
This was more of a sanity check though it's possible someone does `self.__dict__["secret_module"] = nn.Linear()`